funnel/views/email.py

```diff
@@ -122,6 +122,23 @@ def send_password_reset_link(email, user, token):
     send_email(subject, [(user.fullname, email)], content)
 
 
+@signals.proposal_submitted.connect
+def send_email_for_proposal_submission(proposal):
+    for editor in proposal.project.active_editor_memberships:
```

Comment:
I think there is an `active_editors` proxy that gives you `editor.user` directly. This approach is fine too, but the term is `membership`, not `editor`. Roles are adjectives in the backend and only represented as nouns in the UI.